databricks/sdk/mixins/compute.py

```diff
@@ -203,16 +209,25 @@ def select_node_type(self,
             return nt.node_type_id
         raise ValueError("cannot determine smallest node type")
 
-    def ensure_cluster_is_running(self, cluster_id: str):
+    def ensure_cluster_is_running(self, cluster_id: str) -> compute.ClusterDetails:
```

Comment:
why do you need cluster details to be returned? this will have to be adjusted across all SDKs

Reply:
It doesn't have to 🤷‍♂️ Just seemed more useful given that we have it than returning `None`. 

Do you want me to revert that commit?

Reply:
For now, yes.

Reply:
@judahrand could you address this comment?

Reply:
Sorry! I'd forgotten to follow up on this. Done now.